app/presenters/content_item/contents_list.rb

```diff
@@ -49,6 +51,23 @@ def first_item_content
       first_item_text.length
     end
 
+    def first_item_has_long_table?
+      table = find_table
+      return unless table.present?
+
+      row_count = table.css('tr').count
+      row_count > TABLE_ROW_LIMIT
+    end
+
+    def find_table
```

Comment:
I think for clarity this should be called find_first_table - I went to check whether it was scoped when I saw it called in the previous method.

Reply:
@maxgds it's a common convention in ruby to suffix methods which return a boolean with `?`
It's also conventional (at least with ActiveRecord) to return an object of some sort with methods prefixed with `find_`.
Given this method returns a boolean and not a table element I think the method name is OK.